packages/contracts/test/dollar/BondingV2.t.sol

```diff
@@ -19,14 +21,16 @@ contract ZeroState is LiveTestHelper {
         uint256 _weeks,
         uint256 _endBlock
     );
-    event RemoveLiquidityFromBond( // cspell:disable-line
+    // cSpell:disable
+    event RemoveLiquidityFromBond(
         address indexed _user,
         uint256 indexed _id,
         uint256 _lpAmount,
         uint256 _lpAmountTransferred,
         uint256 _lprewards,
```

Comment:
I understand now why you disabled cspell within certain lines. It is because you believe that these methods are intended to interact with already deployed contracts. However, we will be doing a full and proper redeploy of the entire protocol once things are all settled so you should fix these.

Reply:
Lets not comment anything in solidity files for cspell check, eslint check, tslint check. you can just ignore the whole file if no way.  @3commascapital . I

Reply:
@0xcodercrane ok in your opinion if i modify properties?

Reply:
i decided against modifying properties

Reply:
I agree to modify the property names

Reply:
i'll finish this up quick

Reply:
got this error during last commit for a bunch of sol files: 
```
[warn] Ignored unknown option { explicitTypes: "always" }.
[error] No parser could be inferred for file: packages/contracts/src/dollar/BondingV2.sol
```
used no-verify flag to get around